src/vni-manager.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * File: vni-manager.js
+ */
+var _ = require('underscore');
+
+var inputStates = require('./input-states');
+
+var DEFAULT_VNID = '';
+
+/**
+ * Manage the node's vni's by vnid.  If no vnid is provided the IIP VNID will be used.
+ * If the vni for the specified vnid does not exist, a new one will be created.
+ *
+ * @this the component node instance unless otherwise specified on an API
+ * @param facade a component facade
+ * @return the component facade, extended to include the vni interface
+ * 
+ * Usage examples: 
+ *     var vni = this.nodeInstance.vni(); // retrieve the default VNI 
+ *     var vni = this.nodeInstance.vni(vnid); // retrieve the vni associated with the vnid
```

Comment:
Similarly this should be:
<pre>
var vni = node.vni(vnid); // retrieve the vni associated with the vnid
</pre>